app/src/main/java/com/example/demoappforfirebase/Fragment/ChatFragment.kt

```diff
@@ -60,6 +62,29 @@ class ChatFragment : BaseFragment() {
         chatVM = ViewModelProvider(requireActivity()).get(ChatViewModel::class.java)
         preferencesHelper = PreferencesHelper(requireContext())
         fragmentHelper = FragmentHelper(requireActivity())
+        database = FirebaseDatabase.getInstance().reference
+        val otherUserQuery = database.child("Users").child(otherUserId)
+        otherUserQuery.addListenerForSingleValueEvent(object : ValueEventListener {
+            override fun onDataChange(dataSnapshot: DataSnapshot) {
+                chatVM.otherUser = dataSnapshot.getValue(User::class.java)!!
+            }
+
+            override fun onCancelled(error: DatabaseError) {
+                TODO("Not yet implemented")
+            }
+        })
+
+        val userQuery = database.child("Users").child(preferencesHelper.getUserId())
```

Comment:
mozda bi mogao da se napravi current user objekat u nekom od VM-ova, jer se koristi na puno mesta, pa da se to refaktorise
Ako ti se ne radi sada, samo ga zapisi kao task